package.json

```diff
@@ -25,7 +25,8 @@
 		"prettier": "^3.3.3",
 		"prism-react-renderer": "^2.4.0",
 		"react": "^18.2.0",
-		"react-dom": "^18.2.0"
+		"react-dom": "^18.2.0",
+		"yarn": "^1.22.22"
```

Comment:
@palisadoes I have run `npm yarn install` in development so this file has been changed please have a look.